lib/tooltip_widget.dart

```diff
@@ -236,28 +257,28 @@ class ToolTipWidget extends StatelessWidget {
   Widget _getArrow(contentOffsetMultiplier) {
     final contentFractionalOffset = contentOffsetMultiplier.clamp(-1.0, 0.0);
     return Positioned(
-      top: isArrowUp ? position.getBottom() : position.getTop() - 1,
-      left: position.getCenter() - 24,
+      top: ToolTipWidget.isArrowUp ? widget.position.getBottom() : widget.position.getTop() - 1,
+      left: widget.position.getCenter() - 24,
       child: FractionalTranslation(
         translation: Offset(0.0, contentFractionalOffset),
         child: SlideTransition(
           position: Tween<Offset>(
             begin: Offset(0.0, contentFractionalOffset / 5),
             end: Offset(0.0, 0.150),
-          ).animate(animationOffset),
-          child: isArrowUp
+          ).animate(widget.animationOffset),
+          child: ToolTipWidget.isArrowUp
               ? Icon(
-                  Icons.arrow_drop_up,
-                  color: tooltipColor,
-                  size: 50,
-                )
+            Icons.arrow_drop_up,
```

Comment:
should apply condition to the icon property only. Thus we can avoid 3 duplicated lines.

Reply:
Added check inside icon property and altered icons